rest/admin_api.go

```diff
@@ -1304,7 +1304,10 @@ func (h *handler) handleGetRawDoc() error {
 			return err
 		}
 	}
-
+	base.Audit(h.ctx(), base.AuditIDDocumentMetadataRead, base.AuditFields{
```

Comment:
I think this should also emit a standard document read event when includeDoc is true, right?

Reply:
I don't know if we wanted to issue both here or if reading metadata implies that you are reading the document as well.